src/components/Components/CandidateSection/CandidateSection.scss

```diff
@@ -0,0 +1,40 @@
+.CandidateSection {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 96px 126px;
+    gap: 10px;
+    height: 992.38px;
+    background: var(--color-white-lilac);
+
+    &__wrapper__title {
+        height: 43px;
+        font-style: normal;
+        line-height: 133%;
+        color: var(--color-bunting);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+
+        &__rec {
+            width: 24px;
+            height: 8px;
+            background: #0066FF;
+            border-radius: 4px;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+        }
+    }
+
+    &__wrapper__viewmore {
+        position: absolute;
+        top: 16%;
+        right: 10%;
+
+        &__text {
+            color: #7D7CAC;
```

Comment:
here also will be a custom css variable, i've added it in my last PR, --color-text (but it is not merged yet)